app/src/green/res/xml-v25/shortcuts.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shortcuts xmlns:android="http://schemas.android.com/apk/res/android">
+    <shortcut
+        android:icon="@mipmap/ic_shortcut_compose"
+        android:shortcutId="com.keylesspalace.tusky.Compose"
+        android:shortcutLongLabel="@string/compose_shortcut_long_label"
+        android:shortcutShortLabel="@string/compose_shortcut_short_label">
+
+        <intent
```

Comment:
I don't understand why there are two `Intent`s here, I'm really not familiar with shortcuts.

Reply:
They open both so you see the splash screen for a short time and when you close the compose screen you are back at the main screen. Not sure if thats the best behavior, maybe we should just open `ComposeActivity` only 🤔